diff --git a/node_modules/@angular-devkit/build-angular/src/dev-server/index.js b/node_modules/@angular-devkit/build-angular/src/dev-server/index.js
index 6c25dd8..0564dc0 100644
--- a/node_modules/@angular-devkit/build-angular/src/dev-server/index.js
+++ b/node_modules/@angular-devkit/build-angular/src/dev-server/index.js
@@ -43,6 +43,7 @@ const devServerBuildOverriddenKeys = [
     'verbose',
     'deployUrl',
 ];
+const interfaces = require('os').networkInterfaces(); // 在开发环境中获取局域网中的本机iP地址
 async function createI18nPlugins(locale, translation, missingTranslation) {
     const plugins = [];
     // tslint:disable-next-line: no-implicit-dependencies
@@ -202,10 +203,20 @@ function serveWebpackBrowser(options, context, transforms = {}) {
             });
             if (first) {
                 first = false;
+              let IPAddress = '';
+              for(let devName in interfaces){
+                var iface = interfaces[devName];
+                for(let i=0;i<iface.length;i++){
+                  let alias = iface[i];
+                  if(alias.family === 'IPv4' && alias.address !== '127.0.0.1' && !alias.internal){
+                    IPAddress = `http://${alias.address}:${buildEvent.port}`;
+                  }
+                }
+              }
                 context.logger.info(core_1.tags.oneLine `
               **
               Angular Live Development Server is listening on ${options.host}:${buildEvent.port},
-              open your browser on ${serverAddress}
+              open your browser on ${serverAddress} or ${IPAddress}
               **
             `);
                 if (options.open) {
